pkg/internal/local/klient/docker.go

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package klient
+
+import (
+	"github.com/cloudoperators/greenhouse/pkg/internal/local/utils"
+)
+
+// BuildImage - uses docker cli to build an image
+func BuildImage(img, platform, dockerFilePath string) error {
+	return utils.Shell{
+		Cmd: "docker build --platform ${platform} -t ${img} ${path}",
```

Comment:
Could we not call the docker client to trigger the image build ?https://github.com/moby/moby/blob/master/client/image_build.go

Reply:
We would have to send the tar of the whole repo if we use Docker API. This seemed simpler 😅